pkg/dbop/kill.go

```diff
@@ -22,9 +23,16 @@ func (o *operator) KillConnections(ctx context.Context) error {
 			continue
 		}
 
-		if _, err := o.db.ExecContext(ctx, `KILL CONNECTION ?`, p.ID); err != nil {
+		if _, err := o.db.ExecContext(ctx, `KILL CONNECTION ?`, p.ID); err != nil && !isMySQLError(err, 1094) {
```

Comment:
Could you add a comment to describe what is 1094?
https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html

> Error number: 1094; Symbol: [ER_NO_SUCH_THREAD](https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_no_such_thread); SQLSTATE: HY000

>
> Message: Unknown thread id: %lu

Reply:
Thanks. I renamed the fnction and added comment.